projects/addon-table/components/table/th/th.component.ts

```diff
@@ -41,6 +41,10 @@ export class TuiThComponent<T extends Partial<Record<keyof T, any>>> {
     @HostBinding('class._sticky')
     sticky = this.options.sticky;
 
+    @Input()
+    @HostBinding('attr.requiredSort')
```

Comment:
Why do we need attribute binding?

Reply:
I think it was added by mistake in the original PR and we didn't notice it :) Let's remove it

Reply:
Yeah, it was in the original PR, so I moved it to v3, too.